lib/api/spellbook-api.ts

```diff
@@ -63,7 +63,13 @@ function formatApiResponse(
 
 export default function lookupApi(): Promise<FormattedApiResponse[]> {
   if (useCachedResponse) {
```

Comment:
I think we also want to add something like this to ensure the server rendering isn't slowed down.

```suggestion
  if (useCachedResponse) {
    if (process.server) {
      return useCachedResponse;
    }
```